Assets/Scripts/System/Classes/AttractItem.cs

```diff
@@ -0,0 +1,87 @@
+﻿using UnityEngine;
+using System;
+using UnityEngine.UI;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+[System.Serializable]
+public class AttractItem {
+
+    public enum AttractItemType { Video, Image, Text }
+    public AttractItemType type;
+
+    public Sprite sprite;
+    public string text;
+
+    public string pathToItem;
+    public float displayTime = 10;
+
+    public bool voidItem = false;
+
+    /// <summary>
+    /// Constructs an AttractItem to be used later.
+    /// </summary>
+    /// <param name="pathToItem">Path to file, must be absolute.</param>
+    /// <returns>If the construction was successful.</returns>
+    public AttractItem (string filePath)
+    {
+        pathToItem = filePath;
+
+        if (pathToItem.Contains("mp4"))
```

Comment:
You can get the extension with `string ext = Path.GetExtension(filePath);`

Reply:
👍